8.0.0.md

```diff
@@ -0,0 +1,30 @@
+# Parse Server 8 Migration Guide <!-- omit in toc -->
+
+This document highlights specific changes that require a longer explanation. For a full list of changes in Parse Server 8, please refer to the [changelog](https://github.com/parse-community/parse-server/blob/alpha/CHANGELOG.md).
+
+---
+
+- [Invalid Link Page Changes](#invalid-link-page-changes)
+
+---
+
+## Invalid Link Page Changes
+
+In Parse Server 8, the invalid link page will no longer provide the `username` URL parameter. Instead, the URL parameter will now be `expiredToken`. This change affects how expired verification emails are handled.
```

Comment:
Do we also need to update `public/email_verification_link_expired.html`?

Reply:
Good spot!